src/components/elements/input/SsnInput.tsx

```diff
@@ -0,0 +1,132 @@
+import { TextField, TextFieldProps } from '@mui/material';
+import { Box } from '@mui/system';
+import { isNil, padEnd, padStart } from 'lodash-es';
+import { useMemo } from 'react';
+
+import LabelWithContent from '../LabelWithContent';
+import MultiFieldInput from '../MultiFieldInput';
+
+type SsnInputProps = {
+  onChange?: (value: string) => any;
+  onlylast4?: boolean;
+} & Omit<TextFieldProps, 'onChange'>;
+
+const SsnInput = ({
+  inputProps,
+  InputProps,
+  value,
+  label,
+  onChange,
+  helperText,
+  onlylast4 = false,
```

Comment:
I don't think we have a use case for this, but seems fine to keep. It looks like the placeholders are cut off, but we can wait to worry about that until/if we use this variant

<img width="227" alt="Screen Shot 2022-12-23 at 1 02 13 PM" src="https://user-images.githubusercontent.com/5333982/209386214-4a556bd8-cbae-4873-bba6-8272f2f07986.png">

Reply:
Yeah, I noticed that, sounds good.